modules/appeals_api/lib/appeals_api/upload_error.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class UploadError < StandardError
+    attr_accessor :code
+    attr_accessor :detail
+
+    # DOC1xx errors: client errors, invalid submissions
+    DOC101 = 'Invalid multipart payload'
```

Comment:
Should we refactor these into a more central mail specific concern so that they can be shared between Benefits Intake and Appeals or any other Central Mail direct system in the future?

Reply:
done!